RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
```

Comment:
I added that we publish to npm and ruby gems, had a hard time writing the point of releasing, feel free to take a try